test/Yoti.Auth.Tests/ShareUrl/Policy/WantedAttributeBuilderTests.cs

```diff
@@ -12,13 +14,150 @@ public class WantedAttributeBuilderTests
         [TestMethod]
         public void BuildsAnAttribute()
         {
+            SourceConstraint sourceConstraint = new SourceConstraintBuilder()
+                .WithDrivingLicense()
+                .Build();
+
             WantedAttribute result = new WantedAttributeBuilder()
                 .WithName(_someName)
                 .WithDerivation(_someDerivation)
+                .WithConstraint(sourceConstraint)
                 .Build();
 
             Assert.AreEqual(_someName, result.Name);
             Assert.AreEqual(_someDerivation, result.Derivation);
```

Comment:
Should we check that result.Constraints has a size of one here?

Reply:
Might as well! Added in 6af9d45